addons/l10n_eu_service/models/res_company.py

```diff
@@ -69,6 +69,7 @@ def _map_eu_taxes(self):
                                 'type_tax_use': 'sale',
                                 'description': "%s%%" % tax_amount,
                                 'tax_group_id': self.env.ref('l10n_eu_service.%s' % tax_group_fid).id,
+                                'country_id': company.account_fiscal_country_id.id,
```

Comment:
It would be better if this field was a compute editable one...

```
@api.depends('company_id')
def _compute_country_id(self):
    for tax in self:
        tax.country_id = tax.company_id.account_fiscal_country_id
```
instead of
```
    @api.model
    def default_get(self, fields_list):
        # company_id is added so that we are sure to fetch a default value from it to use in repartition lines, below
        rslt = super(AccountTax, self).default_get(fields_list + ['company_id'])

        company_id = rslt.get('company_id')
        company = self.env['res.company'].browse(company_id)

        if 'country_id' in fields_list:
            rslt['country_id'] = company.account_fiscal_country_id.id
```

to late to change it in stable version :/

Reply:
@smetl thanks. 
I'm not sure why the `default_get` was not applying. I agree, compute would be better - provided it is editable for foreign taxes with a VAT number on the Fiscal position.
For now, this just ensures we don't get unusable taxes created.

let me know if I should change something in master